src/main/resources/compile_invalids_wrapper_fail_on_error.sql

```diff
@@ -104,12 +104,15 @@ begin
 end;
 /
 
+set heading off
```

Comment:
do the same in `compile_invalids_wrapper_not_fail_on_error.sql`

Reply:
Done